src/components/Rule.js

```diff
@@ -32,7 +33,7 @@ let Rule = ({
           }}
           {...hover(state.lens.ruleHover)}
         >
-          <FilterContents {...{ node, root, tree }} />
+          <FilterContents {...{ node, root, tree, fields }} />
```

Comment:
We should think carefully about passing things down the hierarchy because every time we add something new, we have to modify a few different things (which will only increase over time, e.g. when we introduce `RefiningGroup`). This was a pain point in the previous version that we should try to solve, e.g. with something like the `Provider` pattern in react or even just generically passing down all args